source/world_builder/features/subducting_plate_models/temperature/mass_conserving.cc

```diff
@@ -226,6 +231,8 @@ namespace WorldBuilder
         {
 
           const double distance_from_plane = distance_from_planes.distance_from_plane;
+          const int plate_model_sommation_number = 100; // for the plate model
```

Comment:
spelling is incorrect, should be summation with a 'u': replace parameter name with plate_model_summation_number.